articles/performance-optimization-techniques-for-node-js/index.md

```diff
@@ -78,7 +72,7 @@ The goal of the query is to pull the required records from your database. The WH
 
 - Use LIMIT to sample the query results
 
-LIMIT will return only the specified number of records. While using LIMIT, ensure that the results are desirable and meaningful when a limit is imposed on the dataset. For example, if our table Customer has 500 records and we only need the first 100 records, LIMIT will be an efficient way to sample out the desirable results by avoiding the selection of the extra 400 records.
+LIMIT will return only the specified number of records. While using LIMIT, ensure that the results are desirable and meaningful when a limit is imposed on the dataset. For example, if our table Customer has 500 records and we only need the first 100 records. LIMIT will be an efficient way to sample out the desirable results by avoiding the selection of the extra 400 records.
```

Comment:
I don't know why you replaced the comma with a period here. This sentence was perfectly fine before, but now you have two incomplete sentences.